### PR TITLE
Added post referrers stats API

### PR DESCRIPTION
**Pull request by SimonBackx**
closes https://github.com/TryGhost/Team/issues/1942

- Added data fixtures for referrers
- Added new endpoint to fetch referrer stats for a given post: `/stats/referrers/posts/:id`
- Added new ReferrersStatsService, responsible for calculating referrer stats

### CODE REVIEW

File: ghost/stats-service/lib/referrers.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    }
+
+    /**
+     * Return a list of all the attribution sources for a given post, with their signup and conversion counts
+     * @param {string} postId 
+     * @returns {Promise<AttributionCountStat[]>}
+     */
+    async getForPost(postId) {
+        const knex = this.knex;
+        const sigupRows = await knex('members_created_events')
```
Comment (allouis) [inline ghost/stats-service/lib/referrers.js:17]: Lil typo here - should be `signupRows`

**Review by allouis — COMMENTED** [review]
Looks good - small comment!

Is the TODO part of another issue? Should we leave the method out all together for now?

**Comment by SimonBackx** [comment]
Yup, Rish is working on the todo here: https://github.com/TryGhost/Ghost/pull/15449 We can probably delete the todo when we merge both changes

File: ghost/stats-service/lib/referrers.js (hunk 1 above)
Comment (SimonBackx) [reply ghost/stats-service/lib/referrers.js]: ✅

**Pull request merged**